gplugins/lumerical/convergence_settings.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseModel
+
+
+class ConvergenceSettingsLumericalEme(BaseModel):
+    """
+    Lumerical EME convergence settings
+
+    Parameters:
+        passes: Number of passes / simulations sweeping a convergence parameter before checking for convergence
```

Comment:
what does this mean? Is this a performance optimization (only check for convergence after running x sweeps so as to not slow them down)?

Reply:
or is this the number of runs we consider gauge variance/consistency?